utils/build/docker/python_otel/flask-poc-otel.Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM datadog/system-tests:flask-poc.base-v1
+
+WORKDIR /app
+
+COPY binaries* /binaries/
+RUN echo "1.0.0" > SYSTEM_TESTS_LIBRARY_VERSION
+RUN echo "1.0.0" > SYSTEM_TESTS_LIBDDWAF_VERSION
+RUN echo "1.0.0" > SYSTEM_TESTS_APPSEC_EVENT_RULES_VERSION
+
+COPY utils/build/docker/python/flask /app
+COPY utils/build/docker/python_otel/flask-poc-otel/app.py /app
+COPY utils/build/docker/python_otel/flask-poc-otel/app.sh /app
+
+#TODO RMM: Change docker flask-poc base to fix psycopg2 ( psycopg2-binary is not supported by open telemetry)
```

Comment:
I prefer to add these lines to flask-poc-base on a separate PR. I have to check that the change on docker base doesn't arise any issue on other tests